src/scene/procedural.js

```diff
@@ -592,7 +592,7 @@ function _createConeData(baseRadius, peakRadius, height, heightSegments, capSegm
 function createCylinder(device, opts) {
     // #if _DEBUG
     if (opts && opts.hasOwnProperty('baseRadius') && !opts.hasOwnProperty('radius')) {
-        DeprecatedLog.log('DEPRECATED: "baseRadius" in arguments, use "radius" instead');
+        Debug.deprecated('"baseRadius" in arguments, use "radius" instead');
```

Comment:
Can you lose the surrounding `// #if _DEBUG` here?

Reply:
I could, but then the condition (with an empty statement) would make it to the release build.